### ISSUE TITLE
If I have support for "es" and "es-US" is asked for, fallback is used

### ISSUE THREAD

**Issue by EmmanuelMess** [aissat/easy_localization#205]
If I have support for "es" and "es-US" is asked for, fallback is used ("en") instead of "es". Event if "useOnlyLangCode: true".

**Comment by Overman775** [comment]
@EmmanuelMess iOS?

**Comment by EmmanuelMess** [comment]
@Overman775 Android emulator.

**Comment by Overman775** [comment]
@EmmanuelMess show the initialization code Easy Localiztion

**Comment by EmmanuelMess** [comment]
Not working:
```
    EasyLocalization(
      supportedLocales: [Locale('en', ''), Locale('es', '')],
      path: 'lib/l10n',
      fallbackLocale: Locale('en', ''),
      useOnlyLangCode: true,
      child: DeadlinesApp(deadlineDao),
    )
```
Working:
```
    EasyLocalization(
      supportedLocales: [Locale('en'), Locale('es')],
      path: 'lib/l10n',
      fallbackLocale: Locale('en'),
      useOnlyLangCode: true,
      child: DeadlinesApp(deadlineDao),
    )
```

**Comment by Overman775** [comment]
@EmmanuelMess Yep, same recording Locale syntax like in [docs](https://flutter.dev/docs/development/accessibility-and-localization/internationalization) 

I will fix this bug when a little free time appears, for now you can use the second variant. Thanks for issue 👍

**Comment by Tobivaria** [comment]
Is it possible to map different locales to one local?
E.g. : Locale('de', 'De'), Locale('de', 'CH'), Locale('de', 'AT') which all should use the same translation file.
But for Locale('es', 'MX'), Locale('es', 'ES') I want to have them separated. 

Of course I could create 3 excact json files, but this wouldn't make sense.

**Comment by EmmanuelMess** [comment]
> Is it possible to map different locales to one local?
> E.g. : Locale('de', 'De'), Locale('de', 'CH'), Locale('de', 'AT') which all should use the same translation file.
> But for Locale('es', 'MX'), Locale('es', 'ES') I want to have them separated.
> 
> Of course I could create 3 excact json files, but this wouldn't make sense.

Not related.
